helm-chart/renku/values.yaml

```
@@ -599,7 +609,7 @@ ui:
           ref: 0.6.1
           name: Renku
         - url: https://github.com/SwissDataScienceCenter/contributed-project-templates
-          ref: 0.5.0
+          ref: 0.4.1
```

```suggestion
          ref: 0.5.0
```
This should not change at all. Unless there is something specific form this PR that requires for the community project templates to change. I suspect this is the result of resolving conflicts in the PR.